SU2_CFD/src/fluid/CSU2TCLib.cpp

```diff
@@ -110,6 +110,13 @@ CSU2TCLib::CSU2TCLib(const CConfig* config, unsigned short val_nDim, bool viscou
     ElDegeneracy(0,5) = 5;
     ElDegeneracy(0,6) = 15;
 
+    if (viscous) {
+      mu_ref[0] = 2.125E-5;
```

Comment:
Worth adding a comment with a reference for these values?

Reply:
Good point.   I can add a quick reference.